PublishedModules/Psychokiller1888/Minigames/Minigames.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
 				klass = getattr(lib, game)
 				minigame = klass()
 				self._minigames[game] = minigame
-				self._SUPPORTED_INTENTS += minigame.intents
+				self._INTENTS += minigame.intents
```

Comment:
this line needs your work, which might affect the rest aswell

Reply:
well they could be added by converting the list to a dict that calls the onMessage method of the current game